packages/syft/setup.cfg

```diff
@@ -56,15 +49,24 @@ syft =
     uvicorn[standard]==0.23.2
     fastapi==0.103.2
     hagrid>=0.3
-    matplotlib==3.8.0
-    dm-haiku==0.0.10
     itables==1.6.2
+    argon2-cffi==23.1.0
+    # ===== DOCKER:CACHED:START ===== #
+    numpy>=1.23.5,<=1.24.4
+    pandas==1.5.3
+    pyarrow==11.0.0
+    matplotlib==3.8.0
+    networkx==2.8
+    jaxlib==0.4.18
+    jax==0.4.18
+    opendp==0.8.0
     safetensors==0.4.0
     transformers==4.34.0
     evaluate==0.4.1
+    dm-haiku==0.0.10
     torch==2.1.0
     recordlinkage==0.16
-    argon2-cffi==23.1.0
+    # =====  DOCKER:CACHED:END ===== #
```

Comment:
I'm clubbing these big packages between these `DOCKER:CACHED` hints. backend.dockerfile will pick these up, install them and cache them before `pip install -e syft`, so any file changes will not trigger a re-download unless these are updated explicitly.